@uportal/notification-banner/src/components/NotificationBanner.vue

```diff
@@ -36,9 +36,25 @@ export default {
             type: String,
             default: '/NotificationPortlet/api/v2/notifications'
         },
+        notificationVariant: {
+            type: String,
+            default: 'info'
+        },
+        notificationIcon: {
+            type: String,
+            default: 'info'
+        },
+        notificationIconSize: {
+            type: String,
+            default: '1x'
+        },
```

Comment:
would it make sense to add validators for these to ensure they are one of the known literals?
https://vuejs.org/v2/guide/components-props.html#Prop-Validation